server/service/GithubService.js

```diff
@@ -619,6 +619,24 @@ export class GithubService {
     const issue = await this.fetchPath('GET', url, null, token)
     return issue.labels.map(l => l.name)
   }
+
+  /**
+   * Returns milestone present on an issue/pull request.
+   *
+   * @param user
+   * @param repo
+   * @param number
+   * @param token
+   * @returns {string}
+   */
+  async getIssueMilestone(user, repo, number, token) {
+    const url = API_URL_TEMPLATES.ISSUE
+                                 .replace('${owner}', user)
+                                 .replace('${repo}', repo)
+                                 .replace('${number}', number)
+    const issue = await this.fetchPath('GET', url, null, token)
+    return (issue.milestone == null) ? null : issue.milestone.title
```

Comment:
Wouldn't it be cleaner when you just add a Global API_URL_TEMPLATES.MILESTONES ? With the endpoint like here https://developer.github.com/v3/issues/milestones/ ?

Because .this.fetchPath would then automatically handle the check if a milestone exist, isn't it?

Reply:
The only way to check if a specific PR has milestone set is to use the following url, https://api.github.com/repos/zalando/zappr/issues/532.
The endpoint you mentioned only returns a list of all milestones of a repo, but I don't want to know if a repo has a milestone.

Reply:
Thanks, I was mixing the API's a bit correct.

Regarding this Check, are you expecting that each PR has a direct reference to an issue and therefore to a milestone?

Do you think also checking for a PR connected only to a Milestone (without an issue) is possible to check? It's more a possible edge case, where I'm thinking out loud.

Reply:
`getIssueLabels` does the same thing, and I took it as template, but if you want I can change it to use the PR endpoint.

See https://developer.github.com/v3/issues/#get-a-single-issue why it works
> Note: GitHub's REST API v3 considers every pull request an issue, but not every issue is a pull request. For this reason, "Issues" endpoints may return both issues and pull requests in the response. You can identify pull requests by the pull_request key.

Reply:
I'm fine, I just wanted to understand why you used that endpoint. Besides that your code looks totally fine and straight forward.